README.md

```diff
@@ -40,11 +40,18 @@ To run the tests it is advisable to have a `.env` file at the root of the projec
 The content of `.env` would then look similar to this:
 
 ```bash
+IMOD_COLLECTOR_DEVEL='D:\checkouts\imod_collector_devel'
+IMOD_COLLECTOR_REGRESSION='D:\checkouts\imod_collector_regression'
 METASWAP_LOOKUP_TABLE='D:\checkouts\DSCtestbench\cases\e150_metaswap\f00_common\c00_common\LHM2016_v01vrz'
-METASWAP_DLL='D:\checkouts\imod_coupler\MetaSWAP.dll'
-METASWAP_DLL_DEP_DIR='D:\checkouts\imod_coupler'
-MODFLOW_DLL='D:\checkouts\imod_coupler\libmf6.dll'
-IMOD_COUPLER_EXECUTABLE='imodc' # Specify an absolute path here to use a packaged version of iMOD Couplers
+
+IMOD_COUPLER_EXEC_DEVEL='imodc' # Specify an absolute path here to use a packaged version of iMOD Coupler
+IMOD_COUPLER_EXEC_REGRESSION='${IMOD_COLLECTOR_REGRESSION}/imodc.exe'
```

Comment:
Does the target audience of this ``README.md`` consist purely of our Deltares colleagues, who want to get stuff to work on Team City. Or does it go beyond this? Only people who are familiar with the Team City setup are familiar with ``${IMOD_COLLECTOR_REGRESSION}`` I think. To help the rest, it would be good to add a comment explaining why you would do it that way.